### PR TITLE
Feat/exlm 1575 card

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1575

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://feat-exlm-1575-card--exlm--adobe-experience-league.hlx.live/

### CODE REVIEW

File: scripts/profile/profile-interests.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import Dropdown from '../dropdown/dropdown.js';
+import { htmlToElement } from '../scripts.js';
+import getSolutionByName from '../../blocks/toc/toc-solutions.js';
+
+loadCSS(`${window.hlx.codeBasePath}/scripts/profile-interests/profile-interests.css`);
+
+const options = [
+  {
+    value: 'Beginner',
+    title: 'Beginner',
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:11]: This should come from placeholders

File: scripts/profile/profile-interests.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  const card = document.createElement('div');
+  card.className = `profile-interest-card ${isSelected ? 'profile-interest-card--selected' : ''}`;
+  const { class: solutionInfoClassName } = getSolutionByName(product);
+
+  // Header
+  const header = htmlToElement(`
+        <div class="profile-interest-header">
+            <div class="profile-interest-logo-wrapper">
+                <img class="profile-interest-logo">
+                <span class="profile-interest-logo-text">My Adobe Product</span>
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:37]: This should come from placeholders

File: scripts/profile/profile-interests.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        </div>
+    `);
+
+  const img = header.querySelector('img');
+  img.src = `/icons/solutions/${solutionInfoClassName}.svg`;
+
+  // Content
+  const content = htmlToElement(`
+        <div class="profile-interest-content">
+            <label for="experience-level">Select your experience level</label>
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:49]: Should be added in placeholders

File: scripts/profile/profile-interests.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        </div>
+    `);
+
+  // eslint-disable-next-line no-new
+  new Dropdown(content, 'Beginner', options);
+
+  // Checkbox
+  const checboxCta = 'Select this product';
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:57]: This should come from placeholders

File: scripts/profile/profile-interests.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        <div class="profile-interest-header">
+            <div class="profile-interest-logo-wrapper">
+                <img class="profile-interest-logo">
+                <span class="profile-interest-logo-text">My Adobe Product</span>
+            </div>
+            <h3>${product}</h3>
+        </div>
+    `);
+
+  const img = header.querySelector('img');
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:43]: Can we add this as icons instead of img tag?

File: scripts/profile/profile-interests.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    title: 'Intermediate',
+  },
+  {
+    value: 'Advanced',
+    title: 'Advanced',
+  },
+];
+
+// eslint-disable-next-line import/prefer-default-export
+export async function buildProductCard(container, element, model) {
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:24]: container param isn't being used anywhere in the code.

Comment (nitin-rachabathuni) [reply scripts/profile/profile-interests.js:32]: Wil be useful to style layout in blocks Ruhi, so kept it.

File: scripts/profile/profile-interests.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+try {
+  placeholders = await fetchLanguagePlaceholders();
+} catch (err) {
+  // eslint-disable-next-line no-console
+  console.error('Error fetching placeholders:', err);
+}
+
+const options = [
+  {
+    value: placeholders["profile-exp-level-beginner"],
```
Comment (ruhisingh1) [inline scripts/profile/profile-interests.js:18]: Value shouldn't be translated.

Comment (nitin-rachabathuni) [reply scripts/profile/profile-interests.js:18]: Noted

File: scripts/profile/profile-interests.js (hunk 5 above)
Comment (nitin-rachabathuni) [reply scripts/profile/profile-interests.js]: done done

File: scripts/profile/profile-interests.css [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    border: 1px solid #D6D6D6;
+    background-color: var(--spectrum-gray-50);
+    width: 260px;
+    height: 186px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.profile-interest-card--selected {
```
Comment (noormd08) [inline scripts/profile/profile-interests.css:13]: Please dont use BEM pattern, as we are not following anything in EDS.

File: scripts/profile/profile-interests.js [hunk 9]
```diff
@@ -0,0 +1,97 @@
+import { decorateIcons, loadCSS } from '../lib-franklin.js';
+import Dropdown from '../dropdown/dropdown.js';
+import { htmlToElement, fetchLanguagePlaceholders } from '../scripts.js';
+import getSolutionByName from '../../blocks/toc/toc-solutions.js';
+
+loadCSS(`${window.hlx.codeBasePath}/scripts/profile/profile-interests.css`);
```
Comment (noormd08) [inline scripts/profile/profile-interests.js:6]: Keep this as promise response. There are possibilities that the CSS would be immediately loaded when we import the JS file in other files.

File: scripts/profile/profile-interests.css (hunk 8 above)
Comment (nitin-rachabathuni) [reply scripts/profile/profile-interests.css]: fixed Noor

File: scripts/profile/profile-interests.js (hunk 9 above)
Comment (nitin-rachabathuni) [reply scripts/profile/profile-interests.js]: done Noor

File: scripts/profile/profile-interests.css [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+.profile-interest-checkbox {
+    display: flex;
+    gap: 8px;
+    align-items: center;
+}
+
+.profile-interest-checkbox {
+    margin-top: 16px;
+}
```
Comment (noormd08) [inline scripts/profile/profile-interests.css:26]: Please club with above one.

File: scripts/profile/profile-interests.js (hunk 9 above)
Comment (noormd08) [reply scripts/profile/profile-interests.js:6]: No changes required.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**